pages/login.vue

```diff
@@ -89,11 +92,18 @@ export default {
   justify-content: center;
 }
 .v-form {
-  padding: 10px !important;
+  padding: 1.25rem;
+  letter-spacing: 3px;
```

Comment:
you added a px again 😆 everything else looks great!

Reply:
oh... ⚠️  thanks for checking and spotting! @ermish

- above point to be changed this evening

Reply:
- replaced px with rem